src/aap_eda/core/models/activation.py

```diff
@@ -74,6 +80,46 @@ class Meta:
     status_updated_at = models.DateTimeField(null=True)
     status_message = models.TextField(null=True, default=None)
 
+    def save(self, *args, **kwargs):
+        # when creating
+        if self._state.adding:
+            if self.status_message is None:
+                self._set_status_message()
+        else:
+            if kwargs is None:
```

Comment:
save method supports several parameters, we want to enforce the update_fields param, dont?